### PR TITLE
Features/pktz5-movies-details-view-2

### PR DESCRIPTION
**Pull request by mazeneldeeb9**
finish movies details view

### CODE REVIEW

File: Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                        .padding(.horizontal)
+                        .lineSpacing(0)
+                }
+                Spacer()
+             
+                
+                
+            }.frame(maxWidth: .infinity, maxHeight: .infinity)
+                .ignoresSafeArea()
+                .background(.mainPurple)
```
Comment (MohammedAbdullaiOS) [inline Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift]: dont ignore safe area in order for text to be visible on face id iphones

File: Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                    Image(systemName: "clock.fill").foregroundStyle(.gray)
+                    Text(handler.movie.getDuration())
+                        .foregroundStyle(.gray)
+                }.padding(.horizontal)
+                Text("About film")
+                    .foregroundStyle(.white)
+                    .font(.title3)
+                    .bold()
+                    .padding()
+                ScrollView {
```
Comment (MohammedAbdullaiOS) [inline Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift:27]: make scroll view on all screen not just the overview part

File: Peek-swiftUI/Modules/MoviesDetailsView/Views/MovieChip.swift [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+struct MovieChip: View {
+    let text: String
+    
+    var body: some View {
+        Text(text)
+            .foregroundStyle(.white)
+            .padding(.horizontal, 14)
+            .padding(.vertical, 6)
+            .background(.lightPurple)
+            .clipShape(RoundedRectangle(cornerRadius: .infinity))
```
Comment (MohammedAbdullaiOS) [inline Peek-swiftUI/Modules/MoviesDetailsView/Views/MovieChip.swift:21]: set line limi to one and use a small font

Comment (mazeneldeeb9) [reply Peek-swiftUI/Modules/MoviesDetailsView/Views/MovieChip.swift:21]: done

File: Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift (hunk 1 above)
Comment (mazeneldeeb9) [reply Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift]: done

File: Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift (hunk 2 above)
Comment (mazeneldeeb9) [reply Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift:27]: done

File: Peek-swiftUI/Modules/MoviesDetailsView/Views/MovieChip.swift (hunk 3 above)
Comment (MohammedAbdullaiOS) [reply Peek-swiftUI/Modules/MoviesDetailsView/Views/MovieChip.swift:21]: use a smaller font. you can can use custom system font and give it the size` .font(.system(size: <#T##CGFloat#>))`

File: Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift (hunk 2 above)
Comment (MohammedAbdullaiOS) [reply Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift:27]: make background color modifier on the scrollview itself and not the content only

File: Peek-swiftUI/Modules/MoviesDetailsView/Views/MovieChip.swift (hunk 3 above)
Comment (mazeneldeeb9) [reply Peek-swiftUI/Modules/MoviesDetailsView/Views/MovieChip.swift:21]: done

File: Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        if handler.isLoading {
+            LoadingView()
+        } else if handler.hasError {
+            ErrorView(callAgain: {handler.fetchMovieDetails(for: movieId)},
+                      errorMessage: handler.errorMessage ?? "Something went Wrong")
+        } else {
+            VStack(alignment: .leading) {
+                MovieTopImage(urlString: "https://image.tmdb.org/t/p/w500/\(handler.movie.posterPath ?? "")")
+                ScrollView {
+                    LazyVStack(alignment: .leading) {
```
Comment (MohammedAbdullaiOS) [inline Peek-swiftUI/Modules/MoviesDetailsView/Views/MoviesDetailsView.swift]: no need to use lazy vstack

**Pull request merged**